src/ui/theme/theme.ts

```diff
@@ -36,12 +36,17 @@ const explorerConfig = defineConfig({
 
 const themeConfig = {
   theme: {
+    config: {
+      initialColorMode: 'light',
+      useSystemColorMode: true,
```

Comment:
this causes theme flashing before cookies are set because the server says light but then my system says dark. please remove this